homeworks/html-css/rosyurch/index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<link rel="stylesheet" href="style.css">
+	<title>Weather</title>
+</head>
+<body>
+	<div class="container">
+		<header>
+			<h1>New York, NY</h1>
+			<p id="overcast">Tuesday, April 15th<br>Overcast</p>
+		</header>
+				
+		<div class="current">
+			<img src="../icons/png/iconfinder_weather04_4102315.png" width="70px" height="70px">
```

Comment:
Remove 'px'  from width and height.
example:

`<img width="70" height="70"></img> // width, height in PX`
`<img width="70%" height="70%"></img> // width, height in %`